### ISSUE TITLE
schedule commands for a future date/time

### ISSUE THREAD

**Issue by yetibot-admin** [yetibot/yetibot#150]

**Comment by kennethkalmer** [comment]
This would be really nice to have!

Examples uses could be morning report with the weather at certain locations, reports of how systems performed the previous day, etc.

**Comment by devth** [comment]
@kennethkalmer cool! I've been wanting this feature for a long time anyway. This might just motivate me to build it.

**Comment by cvic** [comment]
We could use 
https://github.com/overtone/at-at
or 
http://docs.caudate.me/hara/hara-io-scheduler.html

**Comment by devth** [comment]
I'm working on this. Playing with `hara-io-scheduler` currently. It uses an expanded 7-component crontab format that allows expressing seconds.

**Comment by devth** [comment]
`cron` is available as of `0.4.28`!

```
!help cron

cron <schedule> <command> # run <command> according to <schedule>.

<schedule> must be a valid crontab string with 7 space-separated arguments:
second minute hour day-of-week day-of-month month year

Note: this is more expressive than standard cron which only supports 5
arguments: minutes hours day-of-month month day-of-week

See docs at http://docs.caudate.me/hara/hara-io-scheduler.html#schedule.
cron start # start all scheduled tasks if not already started
cron remove <task-id> # remove a task by id
cron list # list the configured cron tasks
cron status # show the status of the scheduler
cron stop # stop all scheduled tasks```
